mobile/src/full/java/org/openhab/habdroid/core/GcmMessageListenerService.java

```diff
@@ -186,6 +186,23 @@ private Notification makeSummaryNotification(int subNotificationCount, long time
     }
 
     private NotificationCompat.Builder makeNotificationBuilder(String channelId, long timestamp) {
+        long[] vibrationPattern;
+        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
+        switch (prefs.getString(Constants.PREFERENCE_NOTIFICATION_VIBRATION, "")) {
+            case "short":
```

Comment:
Should probably use the defined string resources here.

Reply:
I used strings here, because of the following error message. The only fix I found is to convert the switch to an if.
````
:mobile:compileFullStableDebugJavaWithJavacC:\Users\user\StudioProjects\openhab.android\mobile\src\full\java\org\openhab\habdroid\core\GcmMessageListenerService.java:192: error: constant string expression required
            case getString(R.string.settings_notification_vibration_value_off):
                          ^
````